tests/integ/sagemaker/jumpstart/utils.py

```diff
@@ -115,6 +116,20 @@ def download_file(local_download_path, s3_bucket, s3_key, s3_client) -> None:
     s3_client.download_file(s3_bucket, s3_key, local_download_path)
 
 
+def get_public_hub_model_arn(hub: Hub, model_id: str) -> str:
+    filter_value = f"model_id == {model_id}"
+    response = hub.list_sagemaker_public_hub_models(filter=filter_value)
+
+    models = response["hub_content_summaries"]
+    while response["next_token"]:
+        response = hub.list_sagemaker_public_hub_models(
+            filter=filter_value, next_token=response["next_token"]
+        )
+        models.extend(response["hub_content_summaries"])
+
+    return models[0]["hub_content_arn"]
```

Comment:
If we only care about the first model arn, do we need to paginate all the responses?

Reply:
can you move this to the unit tests? integ tests are only for when we need to create new resources.

Reply:
@evakravi sorry I didn't get that. this is the utility function to get the Public hub content arn which in turn we use for creating the model reference in the Hub, why do we need to move it to unit tests?